docs/content/guides/dagster-pipes.mdx

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
is another PR adding this to the guides landing page?

Reply:
Yep! Just hadn't gotten to it yet.

Reply:
Will this be marked as Experimental? I don't see any mention of that on the page

Reply:
@PedramNavid Done!